src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -56,7 +54,7 @@ protected EnergyNet(@Nonnull Location l) {
 
     @Override
     public int getRange() {
-        return RANGE;
+        return EnergyNetComponent.getRange();
```

Comment:
you're trying to access this staticly, how is that supposed to work if it can change per component? 🤔